web/components/FcDataTableRequests.vue

```diff
@@ -100,10 +101,7 @@
           Multiple Locations
           <FcTextNumberTotal
             class="ml-1"
-            :k="hasFilters ? item.studyRequestBulk.studyRequests.length : null"
-            :n="hasFilters
-              ? item.studyRequestsTotal
-              : item.studyRequestBulk.studyRequests.length" />
+            :n="item.studyRequestBulk.studyRequests.length" />
```

Comment:
this does mean we're getting rid of the whole "show only matching sub-requests" thing, though tbh I'm not sure how useful that was - will be interesting to get feedback on this change.